pandas/core/indexes/base.py

```diff
@@ -151,6 +151,11 @@ class Index(IndexOpsMixin, StringAccessorMixin, PandasObject):
 
     _engine_type = libindex.ObjectEngine
 
+    _accessors = frozenset(['dt', 'str', 'cat'])
```

Comment:
maybe leave out 'dt' and 'cat' until they are actually added as accessor? (in a next PR I suppose)